packages/Webkul/API/Http/Resources/Catalog/Product.php

```diff
@@ -56,6 +58,8 @@ public function toArray($request)
                     $product->getTypeInstance()->haveSpecialPrice(),
                     core()->currency($product->getTypeInstance()->getSpecialPrice())
                 ),
+            'regular_price'           => data_get($prices, 'regular_price.price'),
```

Comment:
Hi @khatriafaz, one last thing can you remove one space from '=>'. So that all are in same line.

Reply:
Hi, @devansh-webkul, I have removed unnecessary space from the '=>'.